code/game/objects/items/kitchen.dm

```diff
@@ -129,6 +133,15 @@
 	w_class = WEIGHT_CLASS_NORMAL
 	custom_price = 60
 
+/obj/item/kitchen/knife/hunting
```

Comment:
this has a force of 10 at the moment. it looks cool and deadly, maybe give it a force of 12 instead of 10? a bit better than a kitchen knife, not as good as a combat knife